src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -199,4 +204,19 @@ private HearingWrapper createWrapper(HearingRequest hearingRequest) throws GetCa
                 .sscsServiceCode(sscsServiceCode)
                 .build();
     }
+
+    private void sendPartiesNotifiedUpdateRequest(HearingWrapper wrapper) {
```

Comment:
This should return a Response, such as PartiesNotifiedHearingResponse

Reply:
This calls the client method that performs a PUT operation, updating an existing resource. Does it really need to return something? This is also not in the spec.